lua/spectre_common.lua

```diff
@@ -319,6 +314,10 @@ local function forward_to_destination(method, request_uri, request_headers)
     }
 end
 
+local function inject_source_header(request)
+   request.set_header('X-Smartstack-Source', 'spectre.main')
```

Comment:
I don't think you can use `PAASTA_SERVICE` or `PAASTA_SERVICE`. The instances are called main or canary in all environments. I'd also rather have it in the config file rather than assuming this runs of paasta.

What I don't remember is if this is `spectre.main` also in dev and stage or if there we use `spectre.dev` and `spectre.stage`